### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by FG-TUM**
# Description

Some small stuff I stumbled upon...

### CODE REVIEW

**Comment by FG-TUM** [comment]
@SteffenSeckler please review asap because this fixes a bug in the gauss generator that @kruegener will need :roll_eyes:

File: src/autopas/containers/linkedCells/LinkedCells.h [hunk 1]
```diff
@@ -287,12 +287,6 @@ class LinkedCells : public ParticleContainer<ParticleCell, SoAArraysType> {
    */
   std::vector<ParticleCell> &getCells() { return this->_cells; }
 
-  /**
-   * @copydoc getCells()
-   * @note const version
-   */
-  const std::vector<ParticleCell> &getCells() const { return this->_cells; }
-
```
Comment (SteffenSeckler) [inline src/autopas/containers/linkedCells/LinkedCells.h:295]: is this not needed? I'd leave it, as it makes sort of sense...

File: examples/md-flexible/src/Generator.h [hunk 2]
```diff
@@ -83,7 +83,7 @@ void Generator::cubeGauss(autopas::AutoPas<Particle, ParticleCell> &autopas, con
   dummyParticle.setTypeId(object.getTypeId());
   autopasTools::generators::GaussianGenerator::fillWithParticles(
       autopas, object.getBoxMin(), object.getBoxMax(), object.getParticlesTotal(), dummyParticle,
-      object.getDistributionMean(), object.getDistributionMean());
+      object.getDistributionMean(), object.getDistributionStdDev());
```
Comment (SteffenSeckler) [inline examples/md-flexible/src/Generator.h:86]: This change makes `md-flexible.test-sim` (defined in `examples/md-flexible/CMakeLists.txt:59`) fail, as there are really really fast particles.
Quick fix would be to change the stddeviation of that test...

File: src/autopas/containers/linkedCells/LinkedCells.h (hunk 1 above)
Comment (FG-TUM) [reply src/autopas/containers/linkedCells/LinkedCells.h:295]: This is provided by the parent class. Thus this function only shadows an in fact identical function. Now that I see it, we should move the non-const version to the interface too.

File: examples/md-flexible/src/Generator.h (hunk 2 above)
Comment (FG-TUM) [reply examples/md-flexible/src/Generator.h:86]: we probably should not use a randomly generated scenario with deltaT != 0 as a test then ;)

File: examples/md-flexible/CMakeLists.txt [hunk 3]
```diff
@@ -88,24 +90,24 @@ add_test(
...
         md-flexible
-        --container
-        VerletClusterCells
-        --verlet-cluster-size
-        32
+        --container VerletClusterCells
+        --verlet-cluster-size 32
+        --no-end-config
         --no-flops
     CONFIGURATIONS checkExamples
```
Comment (FG-TUM) [inline examples/md-flexible/CMakeLists.txt:108]: so we still need these tests?

Comment (SteffenSeckler) [reply examples/md-flexible/CMakeLists.txt:108]: can go away, I guess...

**Pull request merged**